nailgun/entities.py

```diff
@@ -3913,6 +4110,7 @@ def __init__(self, server_config=None, **kwargs):
             'url': entity_fields.URLField(required=True),
             'location': entity_fields.OneToManyField(Location),
             'organization': entity_fields.OneToManyField(Organization),
+            'environment_id': entity_fields.IntegerField(),
```

Comment:
The intention of this was to pass `environment_id` that is used in path construction for `import_puppetclasses` action.  `SmartProxy` entity knows nothing about `Environment`, according to apidoc it is not possible to get nor set Environment directly for SmartProxy. So trying to set Environment field in constructor caused errors from API, and trying to send it as data caused errors that `Environment` is not serializable.
However, I've found another solution in code  and I'll try to implement it here.